server/spec/autobind_spec.rb

```diff
@@ -682,4 +682,56 @@
     entitlements = @cp.list_entitlements(:uuid => consumer.uuid)
     entitlements.size.should == 1
   end
+
+  it 'should be matched with consumer on auto attach when SLA of current entitlement is layered exempt' do
+    product1 = create_product(random_string('product'),
+                              random_string('product'),
+                              {:attributes => {:support_level => 'Standard'},
+                               :owner => owner_key})
+    product2 = create_product(random_string('product'),
+                              random_string('product'),
+                              {:attributes => {:support_level => 'Layered',
+                                               :support_level_exempt => 'true'},
+                               :owner => owner_key})
+
+    pool1 = create_pool_and_subscription(owner_key, product1.id)
+    pool2 = create_pool_and_subscription(owner_key, product2.id)
```

Comment:
The method is deprecated. We should create pools directly using create_pool method from candlepin_api.rb

Reply:
Yeah, updated to new method.